examples/webdriverio-test/test/specs/test.e2e.ts

```diff
@@ -119,8 +119,6 @@ const tests: Test[] = [
       await tapButton(driver, '~' + CONNECT_BUTTON);
       if (driver.isIOS) {
         await driver.acceptAlert();
-        await tapApp(driver);
-        await tapButton(driver, '~' + CONNECT_BUTTON);
```

Comment:
?

Reply:
e2e ios tests were not adjusted at some point, that is why i made this change